### ISSUE TITLE
{Freeway-Bug}[Settings]: Contrast ratio for 'Selected' list-item is '1.37:1' with it's background, on activating 'Save' or 'Discard changes' buttons.

### ISSUE THREAD

**Issue by Saiteja341** [microsoft/terminal#15150]
### Windows Terminal version

1.18.902.0

### Windows build number

23031.1000

### Other Software

Test environment:
OS: Version 22H2 (OS build 23031.1000)
Application: Windows Terminal
Tool: accessibility insights for windows

### Steps to reproduce

Repro steps:

1. Open the 'windows terminal' application.
2. Log in with valid credentials.
3. Now open the 'Settings' using 'Ctrl+,' (Key may varies from user to user).
4. Navigate to any of the list-item excluding profiles & select it.
5. Change the settings & activate 'Save' or 'Discard changes' buttons.
6. Open accessibility insights for windows & verify the contrast ratio of the selected list-item with the background.

Observation: Issue repro only on activating 'Save' or 'Discard changes' buttons.

User experience:
Users with low vision will feel difficulty in recognizing the text, if contrast ratio is less than '4.5:1' with the background.

Guideline reference:
https://www.w3.org/TR/WCAG21/#contrast-minimum

Attachment:
[Contrast ratio fails for the selected list item, on activating 'Save' or 'discard changes' buttons..zip](https://github.com/microsoft/terminal/files/11189978/Contrast.ratio.fails.for.the.selected.list.item.on.activating.Save.or.discard.changes.buttons.zip)

### Expected Behavior

Contrast ratio for 'Selected' list-item should be greater than or equal to '4.5:1' with it's background, on activating 'Save' or 'Discard changes' buttons.

### Actual Behavior

On activating 'Save' or 'Discard changes' buttons, contrast ratio for 'Selected' list-item is '1.37:1' with it's background which is less than '4.5:1' (Required).

**Comment by zadjii-msft** [comment]
<img width="964" alt="Contrast ratio fails for the selected list item, on activating 'Save' or 'discard changes' buttons" src="https://user-images.githubusercontent.com/18356694/232844517-c993e806-14b8-4d2a-b29e-6697c60a5373.png">

**Comment by zadjii-msft** [comment]
_classic_ XAML. 

* OS in light theme
* app in dark theme
* hit discard

Guess I'll just go plaster more `RequestedTheme` calls everywhere.

**Comment by zadjii-msft** [comment]
* ffda8c4 ✅
* 1825ca1 ❔
* e106c09 ❌

**Comment by zadjii-msft** [comment]
Looks like this was happily accidentally fixed in #15162, and I believe that's in 1.18.1041
